recorder.js

```diff
@@ -33,6 +36,27 @@ function parse_game_stats(){
             }
         });
     });
+    if (team_game) {
```

Comment:
If this is a team game then we (rightly) throw away the work we did to populate `game['winner']` above. Is there a way we could not do that previous work at all in the team game case?

Reply:
I _think_ I've addressed this via some if/else in looping over stats in the next commit, I'll let you be the judge.